### PR TITLE
How to securely connect to your server KB article

### PR DESCRIPTION
**Pull request by AnthonyHakim**
New KB article - also added links to this KB article to 2 other create server KB articles.

### CODE REVIEW

File: Servers/creating-a-new-bare-metal-server.md [hunk 1]
```diff
@@ -8,6 +8,8 @@
 ### Description
 Creating a bare metal server on CenturyLink Cloud follows a similar flow as [creating a virtual server](../Servers/creating-a-new-enterprise-cloud-server.md), with a few exceptions. The steps below demonstrate how to provision new bare metal machines in the CenturyLink Cloud. For specifications on which capabilities are supported for bare metal servers as opposed to virtual machines, you may reference the [Server Comparison Matrix](../Servers/server-comparison-matrix.md) for details. For more information on Bare Metal servers in general, please refer to the [Bare Metal FAQ](../Servers/bare-metal-faq.md).
 
+Once you've gone through this KB article and created a new server, you can follow this Getting Started guide to learn [how to securely connect to your new server.](https://www.ctl.io/knowledge-base/servers/getting-started-how-to-securely-connect-to-your-server/)
```
Comment (chrislittle) [inline Servers/creating-a-new-bare-metal-server.md]: links to other kb's must be created in line with the readme.

File: Servers/creating-a-new-enterprise-cloud-server.md [hunk 2]
```diff
@@ -9,6 +9,7 @@
 <p><strong>Description:</strong>
 </p>
 <p>The heart of the CenturyLink Cloud Enterprise Cloud Platform is the ability to create and manage virtual infrastructure. In this KB article, we demonstrate how to provision new virtual machines in the CenturyLink Cloud.</p>
+<p>Once you've gone through this KB article and created a new server, you can follow this Getting Started guide to learn <a href="https://www.ctl.io/knowledge-base/servers/getting-started-how-to-securely-connect-to-your-server/">how to securely connect to your new server.</a>
```
Comment (chrislittle) [inline Servers/creating-a-new-enterprise-cloud-server.md]: links to other kb's must be created in line with the readme.

File: Servers/getting-started-how-to-securely-connect-to-your-server.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  "date": "12-10-2015",
+  "author": "Anthony Hakim",
+  "attachments": [],
+  "related-products" : [],
+  "contentIsHTML": false,
+  "sticky": false
+}}}
+
+## You have a shiny new virtual server - now what?
+Congratulations! You've just built a new virtual server on CenturyLink Cloud - we assume you have created a new virtual server.  If not, [follow this other guide first](https://www.ctl.io/knowledge-base/servers/creating-a-new-enterprise-cloud-server/), then return to this guide.
```
Comment (chrislittle) [inline Servers/getting-started-how-to-securely-connect-to-your-server.md]: links to other kb's must be created in line with the readme.

File: Servers/getting-started-how-to-securely-connect-to-your-server.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+## Connect to your shiny new virtual server
+In order to connect to your shiny new virtual server, you will need to:
+
+1. Download and install the OpenVPN client
+2. Download and install the certificate/OpenVPN configuration
+3. Launch the OpenVPN client and connect to your VPN
+4. Launch your Remote Desktop or SSH client, and connect to your server
+
+### 1. Download and install the OpenVPN client
+You will use the OpenVPN client to securely connect to your new virtual server.
```
Comment (chrislittle) [inline Servers/getting-started-how-to-securely-connect-to-your-server.md]: steps 1, 2, 3 are all covered in https://www.ctl.io/knowledge-base/network/how-to-configure-client-vpn/;  this seems like duplicating kb/data for no reason.  if there are updates to using openvpn just update the current guide.  once you get past installing openvpn (which is already documented) there is really 1 step to RDP or SSH.  i'm not sure that warrants a kb.  

openvpn is not the only secure way to connect to servers either.  we have IPSEC Tunnels, CNS, IQ (MPLS) etc.  if this is a kb on the many options to connect those missing seem to limit its usefulness.

**Comment by AnthonyHakim** [comment]
Check is failing because this is the file the other two KBs are referring to. Once the pull request is completed, the file (../Servers/getting-started-how-to-securely-connect-to-your-server.md) will be there, so the checks will no longer fail.

**Comment by jim-phillips** [comment]
Actually, checks are failing because of `servers` instead of `Servers` - folder names are case-sensitive

**Comment by AnthonyHakim** [comment]
I changed the case of `servers` to `Servers`.

**Pull request merged**